DOCS-5291 React Native docs improvements (#17815)
DataDog/documentation: content/en/real_user_monitoring/reactnative/_index.md

-   If you use the [`react-native-navigation`][5] library, then add the `@datadog/mobile-react-native-navigation` package and follow the [setup instructions][6].
-   If you use the [`react-navigation`][7] library, then add the `@datadog/mobile-react-navigation` package and follow the [setup instructions][8].

> If you experience any issue setting up View tracking with `@datadog/mobile-react-navigation` you can check out our [example application][16].

Reviewer: ```suggestion
If you experience any issues setting up View tracking with `@datadog/mobile-react-navigation` you can see our [example application][16] as a reference.
```

> **Note:** This solution comes from this [StackOverflow][14] post.

Reviewer: ```suggestion
**Note**: This solution comes from this [StackOverflow][14] post.
```
